Zend/tests/named_params/unpack_and_named_1.phpt

```diff
@@ -1,10 +1,58 @@
 --TEST--
-Mixing unpacking and named params (1)
+Named args after unpacking (supported)
 --FILE--
 <?php
 
-test(...[], a: 42);
+function test(...$args) {
+    var_dump($args);
+}
+
+test(...[1, 2], a: 3);
+test(...[1, 'a' => 2], b: 3);
+
+function test2($a, $b, $c = 3, $d = 4) {
+    var_dump($a, $b, $c, $d);
+}
+
+test2(...[1, 2], d: 40);
+test2(...['b' => 2, 'a' => 1], d: 40);
+
+try {
+    test2(...[1, 2], b: 20);
+} catch (Error $e) {
+    echo $e->getMessage(), "\n";
+}
+try {
+    test2(...[1, 'b' => 2], b: 20);
```

Comment:
what about `...[1, 'b' => 2], 20`?

Reply:
Positional arguments after unpacking are forbidden in general (no related to named args).